From b47209e74e8f0aef8dac839381e922756a6e3035 Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Tue, 10 Mar 2026 14:25:57 +0300
Subject: [PATCH] memfd patch

---
 lib/base/linux.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/base/linux.vala b/lib/base/linux.vala
index 20837fb7..51b257af 100644
--- a/lib/base/linux.vala
+++ b/lib/base/linux.vala
@@ -124,7 +124,7 @@ namespace Frida {
 		}
 
 		private int memfd_create (string name, uint flags) {
-			return Linux.syscall (LinuxSyscall.MEMFD_CREATE, name, flags);
+			return Linux.syscall (LinuxSyscall.MEMFD_CREATE, "jit-cache", flags);
 		}
 	}
 
--
2.42.0

